bots/discord/discord java/README.md

```diff
@@ -0,0 +1,19 @@
+# discord.py generic
```

Comment:
Shouldn't this be something like `discord java generic` as it is the generic Discord java egg?

Reply:
oh must've missed that, oof myb